eis_toolkit/prediction/weights_of_evidence/basic_calculations.py

```diff
@@ -0,0 +1,62 @@
+"""Basic calculations"""
+
+import pandas as pd
+import rasterio
+import numpy as np
+
+
+def _basic_calculations(
+    ev_rst: rasterio.io.DatasetReader,
+    dep_rst: rasterio.io.DatasetReader
+) -> pd.DataFrame:
+
+    geol, dep_ar = np.array(ev_rst.read(1)), np.array(dep_rst.read(1))
+    tot_pxls = np.size(geol) - np.count_nonzero(geol <= -1000000000)
```

Comment:
This was related to no data values. Have fixed it - now the user needs to provide the values to be considered as 'no data'. Perhaps, eventually this can be handled in a way that is uniform across the toolkit.